lib/todo_detail.dart

```diff
@@ -0,0 +1,82 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'providers.dart';
+
+class TodoDetail extends ConsumerWidget {
+  const TodoDetail({super.key});
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final todoApp = ref.read(todoAppProvider);
+    final index = ModalRoute.of(context)!.settings.arguments! as int;
+    return Scaffold(
+      appBar: AppBar(
+        title: const Text(
+          'Todo詳細',
+          style: TextStyle(color: Colors.white),
+        ),
+        backgroundColor: Colors.blue,
+      ),
+      body: Center(
+        child: SizedBox(
+          width: 300,
+          child: Column(
+            mainAxisAlignment: MainAxisAlignment.center,
+            children: [
+              Container(
+                alignment: Alignment.centerLeft,
+                margin: const EdgeInsets.only(bottom: 4.0),
```

Comment:
この辺は基本的に `.0` まで書かなくても大丈夫なので省略でいいです。
```suggestion
                margin: const EdgeInsets.only(bottom: 4),
```